stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/user/internal/DatabaseUserRepository.kt

```diff
@@ -94,9 +101,11 @@ internal class DatabaseUserRepository(
      * @param limit Int.
      * @param offset Int.
      */
-    override suspend fun selectAllUsers(limit: Int, offset: Int): List<User> {
-        return userDao.selectAllUser(limit, offset).map(::toModel)
-    }
+    override suspend fun selectAllUsers(limit: Int, offset: Int): List<User> = userDao
+        .selectAllUser(limit, offset)
+        .map(::toModel)
+        .map { userCache[it.id] ?: it }
```

Comment:
@JcMinarro not sure what this line is for.
We've read users from DB, and now replacing the DB version with Cached versions?